src/board/system76/common/main.c

```diff
@@ -74,17 +74,17 @@ void main(void) {
 #endif
 
     // Allow CPU to boot
-    gpio_set(&SB_KBCRST_N, true);
+    //gpio_set(&SB_KBCRST_N, true);
```

Comment:
Does the EC not have to do all these, or are these disabled for debug/development purposes?